tests/locale/fi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,8 @@
+# the header message is required to turn off the fuzzy bit for the catalog
```

Comment:
The .gitkeep is in `/en/´ (which is touched by the `makemessages` test), this is in `/fi/` (which is for `compilemessages`)